### ISSUE TITLE
Credentials not being accepted

### ISSUE THREAD

**Issue by jacksmolinski** [gophish/gophish#2942]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?:OSXv 0.12.1 Was the version I installed 

Brief description of the issue: Credentials not being accepted after I changed username as password from default

What are you expecting to see happen? :Access granted

What are you seeing happen?:User name or password incorrect

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
Insert code here
```

Please provide any terminal output that may be relevant below:

```
Insert output here
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - 
 * Step 2 - 
 
 I changed my creds from the default, both username and password - I did this on Friday 11/8 and im back at work on Monday 14/8 and my  new creds aren't being accepted. Need some assistance

**Comment by wand3rlust** [comment]
I just tested this and in my case (tested on ubuntu with default x64 binary) the new creds are getting accepted. Have you checked whether the credentials are the same you entered while changing?

**Comment by jacksmolinski** [comment]
Yes the creds used are correct, I wrote them down and it was saved In my last pass.

Just need a way to reset them to default if possible

**Comment by glennzw** [comment]
Hi @jacksmolinski - sorry you're locked out! You'll need to jump into the db to reset your password. If you're using the default sqlite you can run the following command:

```
§ yum install -y sqlite ||apt-get install -y sqlite3 libsqlite3-dev
§ sqlite3 gophish.db 'update users set hash="$2a$10$IYkPp0.QsM81lYYPrQx6W.U6oQGw7wMpozrKhKAHUBVL4mkm/EvAS" where username="admin";
```

This will reset the admin password to 'gophish'

(via https://github.com/gophish/gophish/issues/2408)

**Comment by jacksmolinski** [comment]
Thankyou for this.


Unfortunately I am just a Security consultant and am getting lost on running that command?

Where should I be running it?

Mac OSX btw

**Comment by jordan-wright** [comment]
@jacksmolinski if you're on macOS then you should just need to run the second command from @glennzw. The `sqlite3` one.

Assuming Gophish is running on your mac, you need to run that in a terminal in the same folder your `gophish.db` is located.

**Comment by jacksmolinski** [comment]
Thanks @jordan-wright for that

Whenever I run the script nothing happens in terminal except I get the word "quote" appearing in the line below

Is there a way to reinstall GoPhish back to factory settings and start fresh?

Cheers

Jack

**Comment by glennzw** [comment]
If you delete the `gophish.db` file you'll be back to a fresh start.

Hope that helps.

**Comment by Hosted-EU** [comment]
There’s A New Phishing Tool Out There.
It Can Generate URLs For Phishing Pages.
And Able To Steal Cookies from Phishing Pages To Bypass 2FA 🔥

Watch Video On How It Works : https://youtube.com/watch?v=BJv3jFuJJOw
